helm/cluster-eks/templates/cilium-helmrelease.yaml

```diff
@@ -16,10 +16,10 @@ spec:
       chart: cilium
       # used by renovate
       # repo: giantswarm/cilium-app
-      version: 0.10.0
+      version: 0.14.0-2c233645daf0c4fad59f5553cf41f96ddfcc54bc
```

Comment:
will change once the cilium PR is merged https://github.com/giantswarm/cilium-app/pull/99